### PR TITLE
Mod 9702 add button component to cards

### PR DESCRIPTION
**Pull request by chasls2**
### Author
- [x] Story Book has been rebuilt (new build artifacts in `/docs`)
- [x] Components have been rebuilt (new build artifacts in `/dist`)

👆These can be done by executing `npm run build` and committing the build artifacts.

- [ ] All new components have been exported in `/index.js` `(if applicable)`
- [ ] Design Review Complete
- [ ] Relevant Draft release notes have been updated describing these changes ([use this template](https://github.com/fedspendingtransparency/data-transparency-ui/blob/master/release_notes_template.md) to document your changes) `(if applicable)`
- [ ] Integration Status Updated on relevant documentation page `(if applicable)`
- [ ] `index.d.ts` updated with new prop types defined for new components. `(if applicable)`

### Reviewer
- [ ] Code review

### CODE REVIEW

File: components/cards/CardButton.jsx [hunk 1]
```diff
@@ -42,14 +43,15 @@ const CardButton = ({
     if (onlyPerformAction === true) {
         return (
             <div className="card__button">
-                <button
+                {/* <button
```
Comment (nick-torres) [inline components/cards/CardButton.jsx:46]: should be deleted?

Comment (chasls2) [reply components/cards/CardButton.jsx:46]: yes. ill remove it

**Comment by nick-torres** [comment]
you'll also have to update the version in package.json

File: components/cards/CardButton.jsx [hunk 2]
```diff
@@ -42,14 +43,15 @@ const CardButton = ({
...
                     className={`card__button--secondary ${variantMapper[variant]} ${customClassName}`}
                     aria-label={`${text}`}
                     disabled={disabled}
                     onKeyUp={(e) => handleKeyUp(e)}
                     onClick={action}>
                     {text || children}
-                </button>
+                </button> */}
+                <Button onKeyUp={(e) => handleKeyUp(e)} onClick={action} copy={text || children} buttonTitle={text || children} buttonSize="md" buttonType="primary" backgroundColor="light" />
```
Comment (nick-torres) [inline components/cards/CardButton.jsx:54]: i would think you would need to pass in the buttonType here? are they all primary buttons?

**Comment by nick-torres** [comment]
do you think the intention of this ticket is to replace these with 'Button' too?

`return (
        <div className="card__button">
            {govLink ? (
                <div
                    className={`card__button--secondary ${variantMapper[variant]}`}
                    role="button"
                    tabIndex="0"
                    aria-label={`${text}`}>
                    <a
                        target="_blank"
                        rel="noopener noreferrer"
                        onKeyUp={(e) => handleKeyUp(e)}
                        onClick={action}
                        href={link}>
                        {text}
                    </a>
                </div>
            )
                :
                (
                    <a
                        className={`card__button--secondary ${variantMapper[variant]} ${customClassName}`}
                        role="button"
                        tabIndex={disabled ? "-1" : "0"}
                        aria-label={`${text}`}
                        href={link}
                        onKeyUp={(e) => handleKeyUp(e)}
                        onClick={action}>
                        {text || children}
                    </a>
                )}
        </div>
    );`

**Comment by nick-torres** [comment]
this is what the homepage looks like 

image

**Comment by chasls2** [comment]
yeah, I added the buttons then make sure the styling matches, then Ill get rid of the other button

**Comment by chasls2** [comment]
Most of the styling additional styling is on usas

File: components/cards/CardButton.jsx [hunk 3]
```diff
@@ -38,51 +44,56 @@ const CardButton = ({
...
             <div className="card__button">
-                <button
-                    className={`card__button--secondary ${variantMapper[variant]} ${customClassName}`}
-                    aria-label={`${text}`}
-                    disabled={disabled}
-                    onKeyUp={(e) => handleKeyUp(e)}
-                    onClick={action}>
-                    {text || children}
-                </button>
+                <Button onKeyUp={(e) => handleKeyUp(e)} onClick={action} copy={text || children} buttonTitle={text || children} buttonSize="md" buttonType={variantMapper[variant] === undefined ? "secondary" : variantMapper[variant]} backgroundColor="light" textAlignment="center" />
```
Comment (nick-torres) [inline components/cards/CardButton.jsx:55]: are they always medium sized?

**Review by nick-torres — APPROVED** [review]

**Review by nick-torres — APPROVED** [review]

**Pull request merged**